blobstore-s3/src/lib.rs

```diff
@@ -644,6 +638,7 @@ impl Blobstore for StorageClient {
                 "cannot put zero-length objects".to_string(),
             ));
         }
+        // TODO: make sure put_object takes an owned `PutObjectRequest` to avoid cloning the whole chunk
```

Comment:
Is this something that was intended to make it into this PR?

Reply:
Yes, it was a note that we need to fix it